app/mailers/report_mailer.rb

```diff
@@ -14,6 +14,32 @@ def budget_status
     )
   end
 
+  def annual_report(year, to_email)
+    proposals = Proposal.approved.where(client_data_type: "Ncr::WorkOrder")
+        proposals = proposals.select {|p| p.client_data.fiscal_year == year}
```

Comment:
it might be better to use a different variable name her so we don't get confused -- what about `proposals_for_fiscal_year`?